examples/cereal/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+# ---- Options ----
+
+option(ENABLE_TEST_COVERAGE "Enable test coverage" OFF)
+
+# ---- Dependencies ----
+
+include(../../cmake/CPM.cmake)
+
+CPMAddPackage(
+  NAME cereal
+  VERSION 1.2.2
+  GITHUB_REPOSITORY USCiLab/cereal
+  OPTIONS
+    "SKIP_PORTABILITY_TEST ON"
+    "JUST_INSTALL_CEREAL ON"
+)
+
+# ---- Create binary ----
+
+add_executable(CPMExampleCereal main.cpp)
+target_link_libraries(CPMExampleCereal cereal )
+set_target_properties(CPMExampleCereal PROPERTIES CXX_STANDARD 17)
+
+# ---- Add code coverage ----
+
+if (${ENABLE_TEST_COVERAGE})
```

Comment:
For the sake of minimal examples I would restrict adding test coverage to testing-related examples. The coverage is only there as an illustration for a complete test and not actually used by the CI or anything in this project.

Reply:
Ok sounds good, I wasn't 100% sure about that.